getting-started/kubernetes/vpp/eks.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Install VPP on an EKS cluster
+description: Install Calico with the VPP dataplane on an EKS cluster
+canonical_url: '/getting-started/kubernetes/vpp/eks'
+---
+
+### Requirements
+
+Beforehand, you should have:
+
+* The AWS CLI installed on your system
+* A version of kubectl installed, that support that of your EKS cluster
+* An EKS cluster configured (without Fargate)
+
+### Installing Calico-VPP on a EKS cluster
+
+First, make sure that no nodes are running in you EKS cluster. Deploying with running nodes will disrupt pods that are already running.
+
+Then you'll need to setup you kubectl credentials, and disable the aws-node daemonset that conflicts with our CNI.
+````
+aws eks --region some-region update-kubeconfig --name your-cluster-name
+kubectl -n kube-system delete daemonset aws-node
+````
+You can then apply the yaml for Calico-VPP
+
+````
+git clone --branch release/v0.12.1-calicov3.17.1 git@github.com:projectcalico/vpp-dataplane.git
+cd vpp-dataplane
+kubectl kustomize yaml/overlays/eks/ | kubectl -n kube-system apply -f -
+````
+
+### Using DPDK
+
+We also support using the dpdk driver on EKS, which should provide better performance
+````
+kubectl kustomize yaml/overlays/eks-dpdk/ | kubectl -n kube-system apply -f -
+````
+
+
+### Further configuration
+
+* [More info on how to configure the way vpp consumes its primary interface]({{ site.baseurl }}/reference/vpp/uplink-configuration)
```

Comment:
* [Configure how VPP consumes its primary interface]({{site.baseurl}}/reference/vpp/uplink-configuration)
* [Encrypt node-to-node traffic using IPsec]({{site.baseurl}}/reference/vpp/ipsec)